### ISSUE TITLE
Request to run npm update

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#1223]
## Where to find the issue

https://github.com/corona-warn-app/cwa-website/blob/master/package-lock.json

## Describe the issue

```
npm audit
found 276 vulnerabilities (13 moderate, 262 high, 1 critical) in 1609 scanned packages
  run `npm audit fix` to fix 261 of them.
  15 vulnerabilities require manual review. See the full report for details.
```

It seems that the npm packages were last updated by @svengabr through https://github.com/corona-warn-app/cwa-website/pull/686 on Dec 17, 2020, almost 5 months ago.

## Suggested change

run

`npm update -dev`

(I don't want to do this myself, because I can only do it in a fork. If it is done by somebody from the Open Source Team, then it can be done in a branch of the repository, instead of a branch of a fork of the repository. The first way is better for testing.)

---
Internal Tracking ID: [EXPOSUREAPP-7150](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7150)

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks. Internal Tracking ID: [EXPOSUREAPP-7150](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7150)

**Comment by Ein-Tim** [comment]
npm also shows that there's a new version of npm available. Maybe a noob question, but do I have to update npm locally or is this something that has to be done here?

**Comment by MikeMcC399** [comment]
@Ein-Tim 
>npm also shows that there's a new version of npm available. Maybe a noob question, but do I have to update npm locally or is this something that has to be done here?

GitHub currently uses npm 6.14.12 (see https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md).

Which version of npm are you using locally and under which operating system? You should be OK if you have 6.14.x.

When the issue here about npm update is resolved, it will not update the version of npm. It will only update `package.json` and `package-lock.json`.

**Comment by MikeMcC399** [comment]
@svengabr 
Your PR https://github.com/corona-warn-app/cwa-website/pull/1250 resolves the majority of the issues and 

`npm audit`

now shows
```
found 7 vulnerabilities (3 moderate, 4 high) in 1611 scanned packages
  run `npm audit fix` to fix 2 of them.
  2 vulnerabilities require semver-major dependency updates.
  3 vulnerabilities require manual review. See the full report for details.
```

**Comment by Ein-Tim** [comment]
> Which version of npm are you using locally and under which operating system? You should be OK if you have 6.14.x.

Yes, I have 6.14.12 and npm does not show me the info to update npm anymore after #1250. 

Thank you for your help!

**Comment by MikeMcC399** [comment]
Fixed by #1294.
